### PR TITLE
Implement update cryptocurrency mutation

### PR DESCRIPTION
**Pull request by IoanSava**
Mutation:
```graphql
mutation UpdateCryptocurrency($cryptocurrency: UpdateCryptocurrencyInput!) {
    updateCryptocurrency(updateCryptocurrencyInput: $cryptocurrency) {
        id
        symbol
        description
    }
}
```

Query variables:
```json
{
    "cryptocurrency": {
        "id": "6f9f8e59-d34e-47f6-930a-9155ba071636",
        "description": "New description"
    }
}
```

Result:
```json
{
  "data": {
    "updateCryptocurrency": {
      "id": "6f9f8e59-d34e-47f6-930a-9155ba071636",
      "symbol": "RON-COIN",
      "description": "New description"
    }
  }
}
```

### CODE REVIEW

File: cryptocurrencies-service/src/graphql/schema.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ const schema = buildSchema(`
         website: String
     }
 
-    input UpdateCryptoCurrencyInput {
+    input UpdateCryptocurrencyInput {
```
Comment (IoanSava) [inline cryptocurrencies-service/src/graphql/schema.ts:35]: Typo 🤦‍♂️
I've fixed all occurrences (including the graphql api resources)

File: cryptocurrencies-service/src/services/cryptocurrency-service.ts [hunk 2]
```diff
@@ -46,21 +46,22 @@ const mappingDefinition: MappingDefinition[] = [
 
 export const getCryptocurrencyById = async (id: string): Promise<Cryptocurrency> => {
     const query = `
-        PREFIX doacc: <http://purl.org/net/bel-epa/doacc#>
-        PREFIX rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+        PREFIX doacc:    <http://purl.org/net/bel-epa/doacc#>
+        PREFIX rdf:      <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+        PREFIX elements: <http://purl.org/dc/elements/1.1/>
```
Comment (IoanSava) [inline cryptocurrencies-service/src/services/cryptocurrency-service.ts:51]: Created a dedicated prefix for http://purl.org/dc/elements/1.1/

File: cryptocurrencies-service/src/services/cryptocurrency-service.ts [hunk 3]
```diff
@@ -143,20 +145,22 @@ export const createCryptocurrency = async (cryptocurrency: CreateCryptocurrencyI
...
-            ${cryptocurrency.website     ? `doacc:${id} doacc:website                                 <${cryptocurrency.website}>                                                .` : ""}
+            ${cryptocurrency.description ? `doacc:${id} elements:description "${cryptocurrency.description}"@en                                         .` : ""}
+            ${cryptocurrency.blockReward ? `doacc:${id} doacc:block-reward   "${cryptocurrency.blockReward}"^^<http://www.w3.org/2001/XMLSchema#string> .` : ""}
+            ${cryptocurrency.blockTime   ? `doacc:${id} doacc:block-time     "${cryptocurrency.blockTime}"^^<http://www.w3.org/2001/XMLSchema#integer>  .` : ""}
+            ${cryptocurrency.totalCoins  ? `doacc:${id} doacc:total-coins    "${cryptocurrency.totalCoins}"^^<http://www.w3.org/2001/XMLSchema#string>  .` : ""}
+            ${cryptocurrency.source      ? `doacc:${id} doacc:source         <${cryptocurrency.source}>                                                 .` : ""}
+            ${cryptocurrency.website     ? `doacc:${id} doacc:website        <${cryptocurrency.website}>                                                .` : ""}
```
Comment (IoanSava) [inline cryptocurrencies-service/src/services/cryptocurrency-service.ts:163]: Fixed the formatting

File: graphql-api-resources/schema.graphql [hunk 4]
```diff
@@ -32,7 +32,7 @@ input CreateCryptocurrencyInput {
     website: String
 }
 
-input UpdateCryptoCurrencyInput {
+input UpdateCryptocurrencyInput {
```
Comment (IoanSava) [inline graphql-api-resources/schema.graphql:35]: Typo

**Review by DianaCrainic — APPROVED** [review]

**Pull request merged**